CourseChamp-Frontend/src/router/index.js

```diff
@@ -42,10 +43,15 @@ export default new Router({
       name: 'CreateCourse',
       component: CreateCourse
     },
+    {
+      path: '/viewreview',
+      name: 'ViewReview',
+      component: ViewReview
+    },
     {
       path: '/createreview',
```

Comment:
Same here actually although that's not your PR I hadn't caught it

Reply:
same, all of it will be put in one page in the future, we are just working on seperate features and merging them into once page in the next sprint